docroot/themes/humsci/su_humsci_gin_admin/dist/su_humsci_gin_admin.portal.css

```diff
@@ -1 +1,33 @@
-.content-header{background:transparent;text-align:center}.content-header,.layout-container{max-width:600px;margin:0 auto}.user-login-form h1{display:none}.simplesamlphp-auth-login-link{display:inline-block;font-size:32px;padding:30px 0;position:relative;color:#b1040e}.simplesamlphp-auth-login-link::after{content:'\f054';font-family:'Font Awesome 5 Free';font-weight:900;font-style:normal;font-variant:normal;text-rendering:auto;line-height:1;color:#9e9e9e;position:absolute;top:calc(50% - 10px);font-size:20px;right:-20px}
+.content-header {
```

Comment:
Shouldn't the compiled css minified?

Reply:
@pookmish This was Andres' recommendation as using un-minified CSS to debug styling issues locally much easier without needing to manage multiple CSS compiling profiles based on the environment.
![Screenshot 2025-01-13 at 3 10 37 PM](https://github.com/user-attachments/assets/a6d0c556-b16b-48dd-83f1-1dbc1c0a4603)

Also, because production and stage config are already being managed via config_split they enable CSS aggregation `$config['system.performance']['css']['preprocess'] = TRUE;` which minifies the CSS for us. So no need to minify things before they are sent to Drupal to be minified (again) and aggregated into a single file.

Reply:
Ok, I didn't see the comment and I noticed the diff in the changes. Just making sure that it's intentional. :+1: